identity/src/main/java/com/android/identity/credentialtype/CredentialTypeRepository.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.identity.credentialtype
+
+/**
+ * A class that contains the metadata of Credential Types. The repository is initially empty, but in
+ * the com.android.identity.credentialtype.knowntypes package there are well known credential types
+ * which can be added using the addCredentialType() method. Applications also may add their own
+ * Credential Types.
+ */
```

Comment:
Nit: Whitespace! Generally, each documented item should have a short succinct one-liner (ideally less than 60 characters), then longer text in separate paragraphs. Use blank lines to separate paragraphs, e.g.

```
/**
 * Short succinct explanation of something.
 * 
 * Longer paragraph, bla bla bla
 *
 * @param foo also whitespaces before params.
 */
```

Also elsewhere.

Reply:
Done